common/compositor/va/varenderer.cpp

```diff
@@ -199,6 +199,65 @@ bool VARenderer::Init(int gpu_fd) {
   return ret == VA_STATUS_SUCCESS ? true : false;
 }
 
+bool VARenderer::QueryVAProcFilterCaps(VAContextID context,
+                                       VAProcFilterType type,
```

Comment:
Before HWCService is enabled, I feel we need some tests. You can extend https://github.com/intel/IA-Hardware-Composer/blob/master/tests/apps/jsonlayerstest.cpp to cover all these cases. This should also help us figure out what kind of API need to be exposed at Display level. @XiaosongWei  Should be able to help extend the test case and running them on Linux.

Reply:
@kalyankondapally @XiaosongWei , If possible, how about to merge this patch first? And for test case, we can pull another patch to implement that.

Reply:
@feiwan1 We would need to push both things together so we can see the e2e API usage. You can work on a patch on top of this.

Reply:
@kalyankondapally ， we have synced that @XiaosongWei will help to write API at Display level. And I will rebase my patch to display level api after xiaosong finish it.